doc/check-options.md

```diff
@@ -107,6 +108,29 @@ All checks allow these global options:
   </tbody>
 </table>
 
+### aria-required-parent
+
+<table>
```

Comment:
NBD but it's strange that we're not using Markdown to generate this table 🤷

Reply:
I remember it was because Markdown table syntax wasn't working so well with nested code blocks in the cells with multiline code blocks